wiki/Help_centre/Store/Store_archive/en.md

```diff
@@ -0,0 +1,125 @@
+---
+tags:
+  - store
+  - product
+  - keyboard
+  - tablet
+  - area
+  - device
+  - setup
+  - pendrive
+---
+
+# osu!store and merchandise
```

Comment:
Needs to be a different title, or wiki search will show 2 `osu!store and merchandise`:

Maybe this:
```suggestion
# Store archive
```